cps/editbooks.py

```diff
@@ -350,6 +350,74 @@ def table_get_custom_enum(c_id):
 @edit_required
 def edit_list_book(param):
```

Comment:
I moved the function to edit_book_param() in order to modularize it. This is so it can be used with the '/editselectedbooks/' calls. This essentially makes the change atomic from the client's perspective. Otherwise when bulk editing metadata, each metadata for each book needs an individual rest API call.